cmd/ctr/run.go

```diff
@@ -2,15 +2,14 @@ package main
 
 import (
 	gocontext "context"
-	"syscall"
-
 	"github.com/Sirupsen/logrus"
 	"github.com/containerd/console"
 	"github.com/containerd/containerd"
 	digest "github.com/opencontainers/go-digest"
 	specs "github.com/opencontainers/runtime-spec/specs-go"
 	"github.com/pkg/errors"
 	"github.com/urfave/cli"
+	"syscall"
```

Comment:
The empty line that was there previously made goimports do the right thing and separate std library imports from non-standard imports.